database/index.js

```diff
@@ -0,0 +1,77 @@
+const connection = process.env.DATABASE_URL || 'postgres://postgres:postgres@localhost:5432/analytics';
+
+const knex = require('knex')({
+  client: 'pg',
+  connection,
+});
+
+const generateListingHost = view => (
```

Comment:
If all of these functions are the same as the ones in the `data_generation.js` I would consider pulling them into a separate `helpers` folder. I think it is generally bad practice to have the same function declared in more than one place. Could be wrong though.